vendor_download.sh

```diff
@@ -77,12 +77,13 @@ esac
 
 echo "Downloading asset information from latest tag for architecture '$ASSET_POSTFIX'"
 
-echo "Downloading themis binary from latest release"
+# TODO: Do not merge with this tag forced on
+echo "Downloading themis binary from v1.0.17-beta release"
 THEMIS_RELEASE_JSON=vendor-bins/themis-release.json
 curl -sSL \
     -H "Authorization: token $GITHUB_TOKEN" \
     -H "Accept: application/vnd.github.v3.raw" \
-    https://api.github.com/repos/fossas/themis/releases/latest > $THEMIS_RELEASE_JSON
+    https://api.github.com/repos/fossas/themis/releases/174635703 > $THEMIS_RELEASE_JSON
```

Comment:
Is this leftover test code?

Reply:
Yes, I need to revert this once we merge and cut a release with this PR included: https://github.com/fossas/themis/pull/68